src/features/seller/transactions/components/stats-grid.tsx

```
@@ -0,0 +1,35 @@
+import { StatsCard } from './stats-card';
+
+interface StatsCard {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'StatsCard'. Consider to delete it or rename it.

'StatsCard' is defined here:


(lint/suspicious/noRedeclare)